oidc-api/build.gradle

```diff
@@ -42,10 +44,25 @@ test {
     exclude 'uk/gov/di/authentication/oidc/contract/**'
 }
 
-task contractTest (type: Test) {
+var pactDir = "$rootDir/oidc-api/src/test/pact"
+
+task pactConsumerTests (type: Test, group: "verification") {
     useJUnitPlatform()
     include '/uk/gov/di/authentication/oidc/contract/**'
-    systemProperties['pact.rootDir'] = "$rootDir/oidc-api/src/test/pact"
+    systemProperties['pact.rootDir'] = pactDir
+}
+
+pact {
+    publish {
+        pactDirectory = pactDir
+        pactBrokerUrl = "$System.env.PACT_URL".toString() + "?testSource=" +"$System.env.PACT_BROKER_SOURCE_HEADER".toString()
```

Comment:
This can be simplified a little: https://github.com/govuk-one-login/ipv-core-back/pull/1802/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7